MycroftAI/mycroft-core
# Fix extraction of 0 in extract_number()

## Description
0 was not detected as a valid number when checking if the function should proceed to check for fractions.

## How to test
Verify that this doesn't change any expected behavior. 0 should be equivalent to False if anyone was depending on this.

## Contributor license agreement signed?
CLA [Yes]
